tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+
+
+```javascript
+// Compute phase and amplitude.
+var phase = harmonicTrendCoefficients.select('sin')
+		.atan2(harmonicTrendCoefficients.select('cos'))  
+		// Scale to [0, 1] from radians.  
+  .unitScale(-Math.PI, Math.PI);
+var amplitude = harmonicTrendCoefficients.select('sin')
+		.hypot(harmonicTrendCoefficients.select('cos'))  
+		// Add a scale factor for visualization.  
+  .multiply(5);
+// Compute the mean NDVI.
+var meanNdvi = filteredLandsat.select('NDVI').mean();
+// Use the HSV to RGB transformation to display phase and amplitude.
+var rgb = ee.Image.cat([
+                        phase,
+                        amplitude, 
+                        meanNdvi
+		]).hsvToRgb()
+Map.addLayer(rgb, {}, 'phase (hue), amplitude (sat), ndvi (val)');
+```
+
+> **Question 2**: Upload the resulting map layer and describe its salient features.
+
+---
+
+###  Complex Time Series Modeling
+
+A time series can be decomposed as the sum of sinusoids at different frequencies. The harmonic model presented here can be extended by adding bands that represent higher frequencies and the corresponding `sin()` band for a harmonic component to account for two cycles per year.  
+
+You can look at this GEE [example](https://code.earthengine.google.com/2669122497313113fc4bb81bc8352828) of using multiple sinusoids to build a more complex harmonic model. Note that each year there is a high peak in June and a secondary peak in January - this harmonic model consisting of two sinusoids with separate frequencies and amplitudes is able to account for that. However, the error values in this model are high and the fit is quite inexact. We can see extreme drops in the NDVI value that the model misses, and several peaks each year that do not fit.
+
+![Complex Harmonic Model](./im/im_05_05.png)
+
+More complex harmonic models might not be appropriate due to overfitting - in other words, this model might  provide a false sense of comfort in its explanatory ability. Time Series modelling of remote sensing data is more difficult than many business or scientific contexts due to masked data, missing data, irregular atmospheric conditions and natural variability.
+
+### Exporting Data
+
+Many of you might be more familiar with building statistical models in other languages or tools, such as Python, R or JMP. After all, JavaScript was not built as a natural statistics tool, and being able to work with the data In that case, you'll likely want to export the data for your own analysis. There are several ways to do it, but the simplest method is to click the 'expand into new tab' button next to the chart that contains the data you want to work with (likely the raw NDVI data) - in the new tab, you can click 'Download .csv', which is a data table that you can use with whichever software you prefer.
```

Comment:
Remove:

> After all, JavaScript was not built as a natural statistics tool, and being able to work with the data

...start the next sentence with "In that case".

Just trying to avoid extra text that is a bit of an aside.

Reply:
Updated sentence